Update Guide “camera-triggers” (#275)
komatic5/gdcs2: content/docs/guides/triggers-1/camera-triggers.md

Reviewer: Does it work in update 2.208?

Reviewer 2: no

Reviewer: ```suggestion
Alongside Add is **Snap360** which __rotates the camera’s position to the nearest 360° angle. In other words, it uprights the camera back to its default position__ As of 2.208, however, it seems that this feature does not work. Even so, you can just input 0° and/or 360° if you want to stabilize the camera.
```

The {{< img src="images/GDEmotes/Triggers/CameraStatic.png" class="emote">}} **Static** trigger __fixes the camera to a selected point on the screen.__

{{< img src="https://lh3.googleusercontent.com/d/1KjXTRnDWlPruiL37WUeyvTkdEyNef0rL" >}}

**Target Pos Group ID** is for the object the camera will lock to. *You can only have one object as the center.*

>  **X Only** makes the trigger ignore the Y-axis when moving the camera. **Y Only** does the opposite; ignoring the X-axis.

Reviewer: ```suggestion
**X Only** makes the trigger ignore the Y-axis when moving the camera. **Y Only** does the opposite; ignoring the X-axis.
```

If the center moves via a Move or Rotate trigger, the **Follow** option makes the camera move along with the center. The **Easing** slider to the right modifies the easing rate.

The **Smooth Velocity** option makes the transition to the target adapt to the current cameras velocity using a bezier curve with its **Modifier** Slider modifying the velocity itself.

As the name implies, **Exit Static** disables the trigger and reverts the camera to its previous state. Ticking **Exit Instant** does it in an instant rather than easing to the previous state.

{{< callout context="caution" title="Warning:" icon="outline/info-circle" >}}

> As of 2.208, there is a bug pertaining to the exit static option. When using a {{< img src="images/GDEmotes/Triggers/CameraStatic.png" class="emote">}}static camera with the Follow option enabled, attempting to exit the static camera will force the cameras Y level to be the same as the object originally being followed when entering a portal without Free mode enabled.

Reviewer: ```suggestion
As of 2.208, there is a bug pertaining to the exit static option. When using a static camera with the Follow option enabled, attempting to exit the static camera will force the camera's Y level to be the same as the object originally being followed when entering a portal without Free mode enabled.
```

The reason of this change is because shortcodes break inside callout blocks.

** **

> To fix this, copy the original static camera trigger and place it 0.1 blocks before the exit static camrea trigger, making sure to uncheck the follow option.

Reviewer: ```suggestion
To fix this, copy the original static camera trigger and place it 0.1 blocks before the exit static camera trigger, making sure to uncheck the follow option.
```

# 4: GP Offset Trigger

Meanwhile, the {{< img src="images/GDEmotes/Triggers/GPOffset.png" class="emote">}} **GP Offset** trigger __shifts the player’s position relative to the camera.__
{{< img src="https://lh3.googleusercontent.com/d/1EfMgGaxgxJxu2wXGFpHsbMszc7JQNTBF" >}}

> **X Only** and **Y Only** works exactly like the Offset trigger, where they ignore each other’s axes. Ticking **Don’t Zoom** treats the offset as though no camrea zoom was applied.

Reviewer: ```suggestion
**X Only** and **Y Only** works exactly like the Offset trigger, where they ignore each other’s axes. Ticking **Don’t Zoom** treats the offset as though no camera zoom was applied.
```

{{< img src="https://lh3.googleusercontent.com/d/1eBOFis9BqkGI_bscwHKBYmCmudN8w_UG" >}}

> The checkboxes **Left, Right, Up** and **Down** determine the edge the camera will be fixed to.**Unlock** resets the camera to its normal state.

Reviewer: ```suggestion
- The checkboxes **Left, Right, Up** and **Down** determine the edge the camera will be fixed to. 
- **Unlock** resets the camera to its normal state.
- Unlike the other camera triggers, you can't use a certain easing or modify the move time.
```

Some gamemodes by default have borders at the top and bottom such as the ball, ship, UFO, wave, spider, and swing gamemode. With the {{< img src="images/GDEmotes/Triggers/CameraMode.png" class="emote">}} **Mode trigger**, __you can remove these borders.__ It’s as simple as ticking **Free Mode**.

> {{< img src="https://lh3.googleusercontent.com/d/1DQ41cxpfmbZcbIfhsEjbCFWUd9X8oNG4" >}}

Reviewer: This link is broken. Please replace with a working url.

Some gamemodes by default have borders at the top and bottom such as the ball, ship, UFO, wave, spider, and swing gamemode. With the {{< img src="images/GDEmotes/Triggers/CameraMode.png" class="emote">}} **Mode trigger**, __you can remove these borders.__ It’s as simple as ticking **Free Mode**.

> {{< img src="https://lh3.googleusercontent.com/d/1DQ41cxpfmbZcbIfhsEjbCFWUd9X8oNG4" >}}

Reviewer: ```suggestion
{{< img src="https://lh3.googleusercontent.com/d/1-Z3zKAwOkcqn1t-gFWhZGLkUfjN2uu4X" >}}
```
